tools/packaging/packaging.strings.psd1

```diff
@@ -73,4 +73,50 @@ OsxDistributionTemplate = @'
     <pkg-ref id="com.microsoft.powershell" version="{1}" onConclusion="none">{2}</pkg-ref>
 </installer-gui-script>
 '@
+NuspecTemplate = @'
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <id>{0}</id>
+        <version>{1}</version>
+        <title>PowerShellRuntime</title>
+        <authors>Powershell</authors>
+        <owners>microsoft,powershell</owners>
```

Comment:
Is this right case?

Reply:
According to Contribution guidelines, only Author, Company and Copyright needs to be updated. There is no section for Company in nuspec and these are the defaults generated by nuspec tool.

Reply:
Sorry, I meant **M**icrosoft,**P**owershell